src/optimization.jl

```diff
@@ -34,10 +34,14 @@ urbandemand = initurbandemand(m) # Just here for the parameters
 
 # Only include variables needed in constraints and parameters needed in optimization
 
-paramcomps = [:Allocation, :Allocation, :UnivariateAgriculture, :IrrigationAgriculture, :IrrigationAgriculture, :Transportation, :Market]
-parameters = [:waterfromgw, :withdrawals, :totalareas, :rainfedareas, :irrigatedareas, :imported, :internationalsales]
-constcomps = [:Agriculture, :WaterNetwork, :Allocation, :Market, :Market]
-constraints = [:allagarea, :outflows, :balance, :available, :domesticbalance]
+paramcomps = [:Allocation, :Allocation, :UnivariateAgriculture,:Allocation]
+parameters = [:waterfromgw, :withdrawals, :totalareas,:returns]
+constcomps = [:UnivariateAgriculture, :WaterNetwork, :Allocation,:Allocation,:UnivariateAgriculture,
+:Allocation,:UnivariateAgriculture,:UnivariateAgriculture]
+constraints = [:allagarea, :outflows, :balance,:totaluse,:sorghumarea,
+:returnbalance,:hayproduction,:barleyproduction]
```

Comment:
I think all of these changes need to be removed, because the rest of the code doesn't have the corresponding changes to it.